### DISCUSSION TITLE
Support postcss customization for 2.0?

### DISCUSSION THREAD

**Discussion by lxy-yz** [vuepress/core#577] [Q&A] [answered]
### What

The [v2](https://github.com/vuepress/vuepress-next/blob/main/packages/@vuepress/bundler-vite/src/plugins/createMainPlugin.ts#L76-L85) new bundler doesn't seem to expose postcss customization via user configurations as [v1](https://v1.vuepress.vuejs.org/config/#postcss) does. One of the common use case that customizing would be useful is when using https://tailwindcss.com/. 

Currently I am only able to get tailwindcss working with vuepress 2.0 by opting it out from vuepress build process.

**Comment by Mister-Hope** [comment, marked as answer]
Not sure if you implement it now, but you are free to customize bundlerConfig from theme or plugins.

If you meet any problems editing bundlerConfig, you can reply with more detail.
